views/users/movie.pug

```diff
@@ -0,0 +1,28 @@
+extends ../application.pug
+
+	block content
+		
+		.container
+			h1=movie.title
+				.row
+					.col-xs-8
+						.card.card-block
+							h6=movie.trailer
```

Comment:
put spaces after = when you have a dynamic value injection. Thus 
```
h6= movie.trailer
```

do it for others too